website/docs/zh/guide/advanced/dts.mdx

```diff
@@ -46,7 +46,7 @@ Rslib 默认使用 [TypeScript Compiler API](https://github.com/microsoft/TypeSc
 
 如果你想生成 Bundleless DTS，可以：
 
-- 设置 `dts: true` 或者 `dts: { bundle: false }` 在 Rslib 配置文件。
+- 设置 `dts: false` 或者 `dts: { bundle: false }` 在 Rslib 配置文件。
```

Comment:
`dts: true` means enable DTS generation which default to bundleless DTS. This is totally correct in docs.

Reply:
Ahaha, sorry, my mistake, there was an "or" in the middle. I thought it was the same as the one at the end.